### PR TITLE
Dark mode theme

### PR DESCRIPTION
**Pull request by cuygur**
- day
day

- night
night

- have error in the test

### CODE REVIEW

**Comment by bertacsevercan** [comment]
I don't think a dark mode is necessary, we have to change a lot of things to make this happen.

**Comment by cuygur** [comment]
@bertacsevercan you can respect my work. It's a very nice feature and it was a challenge. I did all. Only test left.

**Comment by bertacsevercan** [comment]
@cuygur I do respect your work. I just meant this wasn't a priority, if you've finished it, we can use it.

**Comment by cuygur** [comment]
@bertacsevercan okay thanks

**Review by bertacsevercan — APPROVED** [review]

**Comment by bertacsevercan** [comment]
There are conflicts!

File: src/components/NavBar/index.jsx [hunk 1]
```diff
@@ -121,6 +121,27 @@ const NavBar = ({ isTesting }) => {
...
+            <>
+              <Menu.Item key="11" className="navbarAdmin">
+                <Link to="/admin">
+                  <Icon
+                    style={{ transform: "scale(2)" }}
+                    component={AdminSvg}
+                  />
+                </Link>
+              </Menu.Item>
+              <Menu.Item key="9" className="navbarLogout">
```
Comment (bertacsevercan) [inline src/components/NavBar/index.jsx]: This key should be 12

File: src/components/NavBar/index.jsx [hunk 2]
```diff
@@ -121,6 +121,27 @@ const NavBar = ({ isTesting }) => {
...
+                <Link to="/">{t("navbar.headers.header3")}</Link>
+              </Menu.Item>
+            </>
+          ) : (
+            <Menu.Item key="8" className="navbarAdmin">
```
Comment (bertacsevercan) [inline src/components/NavBar/index.jsx]: This key should be 11

**Review by bertacsevercan — CHANGES_REQUESTED** [review]

**Comment by cuygur** [comment]
after the second merge I couldn't pass test..

**Pull request closed without merging**